lib/IDE/CodeCompletion.cpp

```diff
@@ -4661,8 +4671,13 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     for (auto Piece : AccessPath) {
       builder.push_back(Ctx.getIdentifier(Piece));
     }
-    AccessFilteringDeclConsumer FilteringConsumer(CurrDeclContext, *this);
-    TheModule->lookupVisibleDecls(builder.get(), FilteringConsumer,
+
+    UsableFilteringDeclConsumer UsableFilteringConsumer(Ctx.SourceMgr,
+        Ctx.SourceMgr.getCodeCompletionLoc(), *this);
```

Comment:
It seems this function is not called for the current module. So this filter is not necessary. Could you just add an assertion instead?

Reply:
Nice catch! Will do